components/elm/src/cpl/lnd_comp_mct.F90

```diff
@@ -586,7 +586,7 @@ subroutine lnd_run_mct(EClock, cdata_l, x2l_l, l2x_l)
     call lnd_import_moab( bounds, atm2lnd_vars, glc2lnd_vars)
 #endif
 
-    call lnd_import( bounds, x2l_l%rattr, atm2lnd_vars, glc2lnd_vars)
+    call lnd_import( bounds, x2l_l%rattr, atm2lnd_vars, glc2lnd_vars, lnd2atm_vars)
```

Comment:
@rljacob @sarich I will need to redo the lnd_import_moab method, because now lnd_import has an extra argument; the strategy will be as before, just copy / paste the lnd_import method, and replace x2l_l references to x2l_lm